sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorProvider.java

```diff
@@ -67,4 +72,28 @@ public Reactor createReactor(String connectionId, int maxFrameSize) throws IOExc
 
         return this.reactor;
     }
+
+    public ReactorExecutor createExecutorForReactor(String connectionId,
```

Comment:
nit: for the sake of consistency, I see that some parameters are on new lines and others together. personally, I like them in one line.

Reply:
Ah.. my bad, fixed it